smoke-test/tests/cypress/cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
```

Comment:
What is this?

Reply:
(do we need it?)

Reply:
this is an example fixture generated by default by cypress- its just showing someone how they might add another fixture